step-definitions/Hotels/Tc-18.js

```diff
@@ -0,0 +1,68 @@
+//TC18
+
+const { Given, Then, When, And } = require("@wdio/cucumber-framework")
+const {expect} = require("chai");
+
+Given(/^I Launch Hotels.com$/, async function () {
+    await browser.url('https://www.hotels.com/')
+});
+
+Then(/^I Click on Travelers$/, async function () {
+    await $('//button[@class="uitk-menu-trigger uitk-fake-input uitk-form-field-trigger"]').click();
```

Comment:
No selector and no custom function on SD pages. All selectors should go to POM. You should create custom function for each action/verification in POM and call those functions here on SD.